frontend/components/expando/component.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC, useEffect, useState } from 'react';
+import { FC, useState } from 'react';
```

Comment:
I'd like to suggest a change to this file to improve its accessibility.

When one of the accordion is closed, the user can still use the keyboard to navigate its children. To prevent this, we could add `visibility: hidden` to the container of the children.

Below, I've added two `visibility` rules:
```tsx
      <motion.div
        className="px-2 -mx-2 overflow-hidden"
        initial={isOpen ? 'open' : 'closed'}
        animate={isOpen ? 'open' : 'closed'}
        transition={{ bounce: 0, duration }}
        variants={{
          open: { height: 'auto', visibility: 'visible' },
          closed: { height: 0, visibility: 'hidden' },
        }}
      >
        {children}
      </motion.div>
```